frontend-react/src/views/Home/index.js

```diff
@@ -3,13 +3,18 @@ import React from 'react';
 import Page from '../../components/Page';
 import useT from '../../utils/translation';
 
+const statement =
+  'The Polish Humanitarian Action is a Polish non-governmental organisation ' +
+  'which operates in Poland and other countries. Its mission is "to make the' +
+  'world a better place through alleviation of human suffering and promotion' +
```

Comment:
missing space at the end of this part and the one above

Reply:
ah, so that's why the key wasn't matching on `react-revolution` branch :) thanks, will fix it